### PR TITLE
PARQUET-432: Complete a todo for method ColumnDescriptor.compareTo()

### PR DESCRIPTION
**Pull request by proflin**
The ticket proposes to consider the case *path.length < o.path.length* in, for method ColumnDescriptor.compareTo().

### CODE REVIEW

**Comment by proflin** [comment]
@rdblue @liancheng would you mind to take a look at this? It's a fairly small, self-explanatory change. Thanks!

File: parquet-column/src/test/java/org/apache/parquet/column/TestColumnDescriptor.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    assertEquals(column("b").compareTo(column("a", "b")), 1);
+
+    assertEquals(column("a", "b").compareTo(column("a")), 1);
+    assertEquals(column("a", "b").compareTo(column("b")), -1);
+
+    assertEquals(column("").compareTo(column("")), 0);
+    assertEquals(column("").compareTo(column("a")), -1);
+    assertEquals(column("a").compareTo(column("")), 1);
+  }
+}
```
Comment (liancheng) [inline parquet-column/src/test/java/org/apache/parquet/column/TestColumnDescriptor.java]: Newline at the end of file please.

**Comment by liancheng** [comment]
Looks good to me except for one minor formatting issue.

File: parquet-column/src/test/java/org/apache/parquet/column/TestColumnDescriptor.java (hunk 1 above)
Comment (proflin) [inline parquet-column/src/test/java/org/apache/parquet/column/TestColumnDescriptor.java]: Addressed the issue. Thanks!

**Comment by proflin** [comment]
Updates: added a new line at the end of file.
Thanks for the comments, @liancheng !

**Comment by liancheng** [comment]
+1

**Comment by liancheng** [comment]
Thanks for fixing this! Merging to master

**Pull request closed without merging**